service/src/IOGroup.cpp

```diff
@@ -89,6 +90,10 @@ namespace geopm
         // service is not active then loading the ServiceIOGroup will
         // fail.
         if (geopm::has_cap_sys_admin()) {
+            // May want to give this higher priority than the non-safe
+            // msr driver once it is considered more stable.
+            register_plugin(CpufreqSysfsDriver::plugin_name(),
+                            CpufreqSysfsDriver::make_plugin);
```

Comment:
Was it intentional to move this within the has_cap_sys_admin() check?
The previous version of this had the register_plugin() call *outside* this check.

Reply:
This was a work around for before pruning was implemented.  There is now pruning!